### PR TITLE
Rackspace provider and centos build for standalone

### PR DESCRIPTION
**Pull request by wallies**

### CODE REVIEW

File: standalone/hieradata/Debian.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+mesos::repo: 'mesosphere'
+docker::version: '1.4.0'
```
Comment (tayzlor) [inline standalone/hieradata/Debian.yaml]: version should live in common.yml

File: .gitmodules [hunk 2]
```diff
@@ -31,3 +31,9 @@
 [submodule "modules/staging"]
 	path = modules/staging
 	url = https://github.com/nanliu/puppet-staging.git
+[submodule "modules/haproxy"]
```
Comment (tayzlor) [inline .gitmodules]: remove haproxy submodule as it is not being used

File: README.md [hunk 3]
```diff
@@ -36,7 +38,7 @@ Includes -
 * Consul, running in server mode (Optional - configure via ```consul_enable:``` param in ```hieradata/common.yml```  )
 
     $ cd standalone
-    $ vagrant up
+    $ vagrant up (ubuntu/centos)
```
Comment (tayzlor) [inline README.md]: we should add a bit more about supported operating systems (few lines)

**Comment by tayzlor** [comment]
Are you planning to add Rackspace and CentOS support for the cluster version (Could be done separately in a follow up issue)?

File: .gitmodules [hunk 4]
```diff
@@ -31,3 +31,9 @@
...
+	path = modules/haproxy
+	url = https://github.com/puppetlabs/puppetlabs-haproxy
+[submodule "modules/concat"]
```
Comment (tayzlor) [inline .gitmodules]: Not sure we need concat either - unless any of your code introduces it (looks like it doesn't) ?

File: README.md (hunk 3 above)
Comment (wallies) [inline README.md]: will add some about supporting operating systems.

Will do a seperate pull request for the cluster.

**Pull request closed without merging**